src/awesome_map/BoundaryInterceptor.js

```diff
@@ -150,6 +154,20 @@ define(function(require) {
          */
         this._easing = options.easing || EasingFunctions.easeOutQuart;
 
+        this._boundarySensitivity = options.boundarySensitivity || DEFAULT_BOUNDARY_SENSITIVITY;
+        /**
+         * Observable for subscribing to scroll events beyond AwesomeMap boundaries.
+         *
+         * @method AwesomeMap#onScrollPastTopBoundary
+         * @param {Function} callback
+         *        Invoked with (sender, {
+         *            boundary: {@link BoundaryTypes}
+         *        })
+         */
+        // @TODO: Does this need to be disposed somewhere?  If so, where?
```

Comment:
They do HAVE dispose functions, though, which in turn dispose their observables.  However, the BoundaryInterceptor does not have a dispose function.  Its parent InterceptorMixin does, which it sort of inherits, but i dont want to override that.  Is there a clean javascript pattern to override-but-still-call-through?

Thinking back to java, i was thinking something like this in the constructor would work (but i dont know whether this is too confusing/magicky for javascript):
```
var super = {
    dispose: this.dispose
}
this.dispose = function() {
    // Blah cleanup stuff in this instance
    super.dispose(); // Parent method
```
I'm pretty sure this would work, but it's very java-y.  To simplifiy it, and make it more javascript-y i was thinking i could take out the super var and just do this:
```
this.dispose = function() {
    // Blah cleanup stuff in this instance
    this.dispose(); // Parent method
```
But i'm not sure how the resolution would work.  When the new dispose function gets called, would the inner this.dispose() point to the function that lived there at the time the function was defined, or the function there at call-time (itself) causing an infinite recursion?

And the bigger question is.....does it matter at all, really?  Are observables prone to memory leaks if not disposed properly?